packages/design-tokens/src/tokens/color.tokens.json

```diff
@@ -1,6 +1,7 @@
 {
   "color": {
     "text-strongest": {
+      "type": "color",
```

Comment:
Why do we need this tokens file? How do you ensure all of these props are also in the `colors.js` file, for example? How can we keep both well-updated?

Reply:
The colors.js file is the build output of this token file. So on any build the js file gets updated.